draft-ietf-tls-tls13.md

```diff
@@ -2746,13 +2761,24 @@ from the resumption master secret:
                             "resumption context", "", Hash.Length)
 ~~~~
 
-The client MAY use this PSK for future handshakes by including
-the ticket value in the "pre_shared_key" extension in its ClientHello
-({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
-suite. Servers may send multiple tickets on a single connection, for
-instance after post-handshake authentication. For handshakes that
-do not use a resumption_psk, the resumption_context is a string of
-Hash.Length zeroes.
+The client MAY use this PSK for future handshakes by including the
+ticket value in the "pre_shared_key" extension in its ClientHello
+({{pre-shared-key-extension}}). Servers MAY send multiple tickets on a
+single connection, either immediately after each other or
```

Comment:
Something about the either/or here is bothering me, though I don't quite have a concrete suggestion. (The potential wrong implication being that only one ticket may be sent after any given specific even.)